keel-core-test/src/main/kotlin/com/netflix/spinnaker/keel/persistence/ResourceRepositoryTests.kt

```diff
@@ -201,28 +202,27 @@ abstract class ResourceRepositoryTests<T : ResourceRepository> : JUnit5Minutests
           context("updating the state again") {
             before {
               tick()
-              // TODO: ensure persisting a map with actual data
-              subject.appendHistory(ResourceDeltaDetected(resource, emptyMap(), clock))
+              subject.appendHistory(ResourceValid(resource, clock))
```

Comment:
This was unrelated, but `ResourceDeltaDetected` has `ignoreRepeatedInHistory` set to `false`, so I changed this to an event that had it set to `true`.